tests/integration/components/scrollable-test.js

```diff
@@ -106,16 +103,19 @@ module('Integration | Component | scrollable', function (hooks) {
         <Behave::Scrollable
           class="MyScrollable"
           style="height: 100px;"
+          as |scrollable|
         >
-          <div style="height: 500px; width: 100px; background: deeppink;"></div>
+          <scrollable.content>
+            <div style="height: 500px; width: 100px; background: deeppink;"></div>
+          </scrollable.content>
         </Behave::Scrollable>
       `);
 
       const scrollable = this.element.querySelector('.MyScrollable');
       const scrollableContent = this.element.querySelector('.Behave-Scrollable-Content');
 
       scrollableContent.scrollTo(0, 50);
-      await settled();
+      await pause();
```

Comment:
What has caused the need to change this?

Reply:
Some tests were flaky and this seemed to resolve the flakiness. 

I found no graceful way of waiting for a scroll event to trigger after scroll, the browsers seems to provide no guarantees.

A custom helper based on `waitUntil` could probably be possible, but awkward to use.